### PR TITLE
Generate Flow files into dist

### PR DESCRIPTION
**Pull request by comp615**
Ok! So after some testing, I was able to get this mostly going. The key it turns out is to tell flow that it needs to key off the module field instead of the main field in package.json.

Luckily, it has a new option for this, however I had trouble figuring this out, so I wanted to try and feature it prominently in the docs. I think this is a great path forward as it will naturally stay in sync going forward. There are some quirks that we can improve internally (i.e. ViewProps is not a closed type, so it allows passing props that are not valid...but it's a start)

### CODE REVIEW

File: packages/docs/src/pages/docs/getting-started/multi-platform.md [hunk 1]
```diff
@@ -65,10 +65,12 @@ module.exports = {
 
 ### Flow
 
-[Flow](https://flow.org) can be configured to understand the aliased module. You may also need to include a custom libdef ([example](https://gist.github.com/paularmstrong/f60b40d16fc83e1e8e532d483336f9bb)) in your config.
+[Flow](https://flow.org) can be configured to understand the aliased module and pull types from Reac Native Web's source code. To do this, you can alias the name as below. You will also need to tell flow to use the `module` field of RNW's `package.json` or it may not find the flow types.
 
 ```yml
 [options]
+module.system.node.main_field=module
+module.system.node.main_field=main
```
Comment (necolas) [inline packages/docs/src/pages/docs/getting-started/multi-platform.md:73]: Are both these lines meant to be here, or just the first?

Comment (necolas) [reply packages/docs/src/pages/docs/getting-started/multi-platform.md:73]: This isn't very well documented but I'm assuming flow picks the first recognized entry if there are several: https://flow.org/en/docs/config/options/#toc-module-system-node-main-field-string

**Comment by necolas** [comment]
Merged into the 0.16-dev branch.

This should also resolve #1684

**Comment by comp615** [comment]
Yes correct, it goes down the list till it finds a matching one. Not sure
from documents if it always puts main in, so included anyways.

**Pull request closed without merging**